server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -144,29 +144,22 @@ export class MetadataExtractionProcessor {
   async extractExifInfo(job: Job<IExifExtractionProcessor>) {
     try {
       const { asset, fileName }: { asset: AssetEntity; fileName: string } = job.data;
-      const exifData = await exifr.parse(asset.originalPath, {
-        tiff: true,
-        ifd0: true as any,
-        ifd1: true,
-        exif: true,
-        gps: true,
-        interop: true,
-        xmp: true,
-        icc: true,
-        iptc: true,
-        jfif: true,
-        ihdr: true,
-      });
+      await exiftool.version().then((version: any) => Logger.debug(`We are running ExifTool v${version}`));
+      const exifData = await exiftool.read(asset.originalPath);
+      Logger.log(exifData);
```

Comment:
We probably want to remove this line, otherwise, it will flood the console

Reply:
We could leave it, but use verbose instead.

Reply:
Ah, yeah or at least changed to debug but probably best to just remove?

Reply:
Can we remove the version log too? If anything, only do it once at startup (in the constructor, instead of for every asset it processes.

Reply:
Removed both of those entirely.